projects/bin-picking-with-vision/application.py

```diff
@@ -0,0 +1,210 @@
+import math
+import numpy as np
+import cv2
+from PIL import Image
+
+from jacobi import Obstacle, Motion, Frame, Planner, Studio, PointCloud, CartesianWaypoint
+from jacobi_vision.drivers import SimulatedCameraDriver
+from jacobi_vision import ImageType, ColorImage
+
+
+class VisionBinPicking:
+    def __init__(self, project_path: str):
+        # Load the planner and get references to the robot and camera
+        self.planner = Planner.load_from_project_file(project_path)
+        self.ur5 = self.planner.environment.get_robot('Universal UR5e')
```

Comment:
```suggestion
        self.robot = self.planner.environment.get_robot()
```
I'd use `self.robot` throughout the code. Also don't need a name as long as it's the only robot.